README.md

```diff
@@ -102,6 +109,13 @@ More information in [wiki page](https://github.com/adireddy/haxe-checkstyle/wiki
 			}
 		},
 		{
+			"type": "ParameterNumber",
+			"props": {
+				"severity": "INFO",
+				"maxParameter": 7
```

Comment:
Same here. Please change `maxParameter` to `max` http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/sizes/ParameterNumberCheck.html